src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -434,6 +443,9 @@ public function sendData($data)
             );
         // If you try to parse an empty response body, error will be thrown.
         } catch (\RunTimeException $e) {
+            // Log any errors.
+            $eventDispatcher->dispatch(Constants::OMNIPAY_REQUEST_ERROR, new ErrorEvent($e));
```

Comment:
for the regular AuthorizeRequest, it only dispatches an error if it throws an exception. Whereas here, it's going to dispatch an error even though a response object will be returned. that seems inconsistent.

Reply:
yeah good point.

Reply:
There's no try catch for `$httpRequest->send()` above. I guess I can emit the error event here. @DanayaMel thoughts?

Reply:
if send can throw an exception, i think that sounds good